metrics-operator/controllers/common/providers/datadog/datadog_test.go

```diff
@@ -276,9 +330,17 @@ func TestEvaluateQuery_EmptyPayload(t *testing.T) {
 		},
 	}
 	kdd := setupTest(apiToken)
-	metric := metricsapi.KeptnMetric{
-		Spec: metricsapi.KeptnMetricSpec{
-			Query: "system.cpu.idle{*}",
+	metrics := [2]metricsapi.KeptnMetric{
```

Comment:
Since we might add more cases across the whole set of tests, I suggest having a slice instead of an array. The performance for testing is not a requirement, hence I would rather having a more easier to use code :) 
```suggestion
	metrics := []metricsapi.KeptnMetric{
```

Reply:
Fixed